pr-build.yml

```diff
@@ -0,0 +1,76 @@
+name: PR Build
+
+on:
+  pull_request:
```

Comment:
Note that this workflow does NOT run on pushes to master! Only on pull requests.